routes/field-data.js

```diff
@@ -117,6 +117,29 @@ module.exports = [
           return accum; }, []
         ));
       });
+    },
+  },
+  {
+    /**
+     * @api {get} /field/ids List of VProMMs ids with field data
+     * @apiGroup Field
+     * @apiName Field ids
+     * @apiDescription Returns a list of VPromms ids with field data
+     * @apiVersion 0.1.0
+     * 
+     * @apiExample {curl} Example Usage:
+     *   curl http://localhost:4000/field/ids
+     * @apiSuccessExample {array} Success-Response:
+     *   ["024BX00040","022BX00029", ...] 
+     */
+    method: 'GET',
+    path: '/field/ids',
+    handler: function (req, res) {
+      knex('field_data_geometries')
+      .select('road_id as id')
+      .then(roads => res(
+        uniq(roads.filter(road => road.id).map(road => road.id)))
```

Comment:
No need to use `uniq`; this is definitely something that should be done on the database side, using `DISTINCT`. http://knexjs.org/#Builder-distinct

And make sure to `WHERE`-filer out null/missing road IDs.